instances/atlas/parcellationEntity/DWMA/DWMA_arcuateFasciculus.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/DWMA_arcuateFasciculus",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+      {
+        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/DWMA_deepWhiteMatter"
+      }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "DWMA_arcuateFasciculus",
+    "name": "arcuate fasciculus",
+    "ontologyIdentifier": null,
+    "relatedUBERONTerm": "http://purl.obolibrary.org/obo/UBERON_0035937"
```

Comment:
This needs to point to the controlledTerms/UBERONParcellation instances.

Reply:
done for all